ENH add information on patches to python (#1838)
conda-forge/conda-forge.github.io: src/maintainer/maintainer_faq.rst

  .. note::

    Even if the maintainer isn't active anymore, we generally like to keep them in the list of maintainers and not remove them, in case they want to take up maintenance at a later date.

.. _mfaq_changes_to_major_projects:

:ref:`(Q) <mfaq_changes_to_major_projects>` **Does ``conda-forge`` ever make significant changes or apply code patches to significant upstream packages?**

>   We generally do not make changes, but there are many notable exceptions and we have no set policy. These changes currently fall into

Reviewer: ```suggestion
  We generally try to avoid changes, but there are many notable exceptions and we have no set policy. These changes currently fall into
```

  a few categories. Upstream projects that violate our community norms or pose significant security risks through their policies may
  be changed so that they can be distributed on ``conda-forge``. In many cases though, these projects are not distributed at all. We
  do employ extensive changes to project build scripts in order to properly build and install projects into conda environments.

>   Finally, in some cases we add, enable, or disable features in specific projects to ensure they are braodly compatible with the

Reviewer: ```suggestion
  Finally, in some cases we add, enable, or disable features in specific projects to ensure they are broadly compatible with the
```

  ``conda-forge`` package set. The set of patches/changes we apply is always located in the feedstock that built the package. We

>   also maintain a list of significant changes for some packages in our documentation

Author: ```suggestion
  also maintain a list of significant changes for some packages in our documentation.
```

>   ``conda-forge`` package set. The set of patches/changes we apply is always located in the feedstock that built the package. We
>   also maintain a list of significant changes for some packages in our documentation.

Author: ```suggestion
  ``conda-forge`` package set. The set of patches/changes we apply is always located in the feedstock that built the package. We
  also maintain a list of important packages with changes in our documentation.
```
